save-backend/build.gradle.kts

```diff
@@ -107,8 +107,8 @@ dependencies {
                 "save-agent, please test them on Linux " +
                 "or put the file with name like `save-agent-*-distribution.jar` built on Linux into libs subfolder."
         )
-        runtimeOnly(files("$buildDir/agentDistro").apply {
-            builtBy(downloadSaveAgentDistroTaskProvider)
+        runtimeOnly(fileTree("$buildDir/agentDistro").apply {
```

Comment:
IIRC @0x6675636b796f75676974687562 has changed this from `fileTree` to `files`.

Reply:
this doesn't work on windows

Reply:
@0x6675636b796f75676974687562